scripts/linters/test_files/invalid_merge_conflict.py

```diff
@@ -20,8 +20,4 @@
 
 from __future__ import annotations
 
-<<<<<<< HEAD
-'this is some content to mess with content to append.'
-=======
 'totally different content to merge later.'
->>>>>>> new_branch_to_merge_later
```

Comment:
Why are you removing this? We have it here to test that our linter will flag unresolved merge conflicts